command/stats.go

```diff
@@ -0,0 +1,18 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/hex337/alex-koin-go/model"
+)
+
+type StatsCommand struct{}
+
+// TODO: Add more stats - coins received, coins transfered, total coins
+func (c *StatsCommand) Run(msg string, event *CoinEvent) (string, error) {
+	user := event.User
+
+	coinCount := model.CoinsCreatedThisWeekForUser(user)
```

Comment:
I wonder if we should have err handling here? eg if the user does not exist? This is where I struggle a bit with gorm...

Reply:
Probably, but I was trying to yolo this. Will add proper error handling.

Reply:
+1 on error handling everywhere, its the golang way. Otherwise lgtm

Reply:
I don't think this will actually error if the user doesn't exist. The user will always exist since we ensure anyone who uses a command exists before going further in the command.

Reply:
But there could be other errors? DB connection, etc? More curious about overall error handling but don't let that block you.